.dockerignore

```diff
@@ -0,0 +1,10 @@
+.git
+test_data
+awsscp
+LICENSE
+Make
+*.md
+**/*.xml
+**/*.json
+**/*.iml
+**/*.yaml
```

Comment:
Can we please have a newlines ad the end of files .... it should be in your editor settings.